src/thunderbots/software/param_server/cfg/Params.cfg

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+import roslib
+PACKAGE = "param_server"
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+"""
+this file contains the parameters used by ai, the current implementation requires 
+the parameter to be defined in this file, as well as in dynamic_parameters.cpp
+with the same parameter name
+
+the value will be overriden by the external configuration, the value provided 
+in dynamic_parameters will only be used when no other value has been set
+"""
+
+gen = ParameterGenerator() # create the parameter generator
+
+#NOTE: format gen.add(name, type, level, description, default_value, min, max)
+#min, max only apply for int/double types
```

Comment:
What do we do for other types? (Not super fussed if we actually support them or not, more just commenting that we should be explicit here.)

Reply:
added a comment